src/components/Avatar/Avatar.tsx

```diff
@@ -9,10 +10,10 @@ export type AvatarProps = {
 
 const Avatar: React.FC<AvatarProps> = ({ src, alt, letter }) => {
   if (!src) {
-    return <div className="avatar avatar_accent">{letter}</div>;
+    return <div className={styles['avatar avatar_accent']}>{letter}</div>;
```

Comment:
так, вот это не должно работать
у тебя в `styles` объект, где ключ -  оригинальное название класса, значение - название класса с хэшом
```suggestion
    return <div className={`${styles.avatar} ${styles.avatar_accent}`}>{letter}</div>;
```